Vic2ToHoI4/Source/V2World/Rebellions/RebelTypes.h

```diff
@@ -0,0 +1,31 @@
+#ifndef REBEL_TYPES_H
+#define REBEL_TYPES_H value
+
+
+
+#include "Parser.h"
+#include "RebelType.h"
+
+
+
+namespace Vic2
+{
+
+class RebelTypes: commonItems::parser
+{
+
+  public:
+	class Factory;
+
+	[[nodiscard]] const auto& getRebelTypesMap() const { return rebelTypesMap; }
+	[[nodiscard]] const auto& getRebelType(const std::string& type) const { return rebelTypesMap.at(type); }
+
+  private:
+	std::map<std::string, RebelType> rebelTypesMap;
```

Comment:
Might it make more sense to just store the ideology? RebelType would still be useful for importing the ideology, but you could save a layer of abstraction later.